lib/seattleflu/id3c/cli/command/etl/redcap_det_swab_n_send.py

```diff
@@ -352,18 +352,11 @@ def specimen_barcode(record: Any) -> str:
         value = barcode
     )
 
-    if not record.get('collection_date'):
-        LOG.warning("Could not create Specimen Resource due to lack of collection date.")
-        return None, None
-
     # YYYY-MM-DD in REDCap
-    collected_time = record['collection_date']
+    collected_time = record['collection_date'] if record['collection_date'] else None
```

Comment:
You can shorten this with `record.get('collection_date')

Reply:
The REDCap API returns empty strings for all values that are not filled so it would never be `None`

Reply:
`record['collection_date'] or None` is the shorter version then. But the long version is also just fine. :-)